content.php

```diff
@@ -12,16 +12,18 @@
 	<!-- singe.php -->
 	<article id="post-<?php the_ID(); ?>" <?php post_class(); ?>>
 		<div class="entry-header">
-			<h1><?php the_title(); ?></h1>
+			<h2><?php the_title(); ?></h2>
 		</div>
 		<div class="entry-meta">
+			<p>
```

Comment:
Maybe can we wrap this into a function and try and reuse it? 
Sorry about my "function" obsession :)

Reply:
Yes, this is a reusable template part. This is fine for now. When we revisit the Blog for the redesign then these can be addressed then.